modules/mono/editor/GodotSharpTools/Project/ProjectGenerator.cs

```diff
@@ -6,6 +6,9 @@ namespace GodotSharpTools.Project
 {
     public static class ProjectGenerator
     {
+        const string CoreApiProjectGuid = "{AEBF0036-DA76-4341-B651-A3F2856AB2FA}";
+        const string EditorApiProjectGuid = "{8FBEC238-D944-4074-8548-B3B524305905}";
```

Comment:
Will those always stay the same? Might be worth documenting where those IDs come from/what they mean with a short comment.

Reply:
These are randomly generated unique IDs. They will stay the same, unlike before when we were generating them every time for the same project.